ui-components/src/page_components/listing/AdditionalFees.tsx

```diff
@@ -6,31 +6,41 @@ export interface AdditionalFeesProps {
   listing: Listing
 }
 
-const AdditionalFees = (props: AdditionalFeesProps) => (
-  <div className="info-card bg-gray-100 border-0">
-    <p className="info-card__title">{t("listings.additionalFees")}</p>
-    <div className="info-card__columns text-sm">
-      <div className="info-card__column">
-        <div className="text-base">{t("listings.applicationFee")}</div>
-        <div className="text-xl font-bold">${props.listing.applicationFee}</div>
-        <div>{t("listings.applicationPerApplicantAgeDescription")}</div>
-        <div>{t("listings.applicationFeeDueAt")}</div>
+const AdditionalFees = (props: AdditionalFeesProps) => {
+  const getDeposit = () => {
+    const min = props.listing.depositMin
+    const max = props.listing.depositMax
+    if (min && max && min !== max) {
+      return `$${min} – $${max}`
+    } else if (min) return `$${min}`
+    else return `$${max}`
```

Comment:
Can you update this to use braces throughout? I think this would be the only case that doesn't, and can you also add the multi-line lint rule https://eslint.org/docs/rules/curly#multi-line.

Reply:
Chatted over slack, created https://github.com/bloom-housing/bloom/issues/1419